app/src/main/java/org/oppia/app/home/HomeActivity.kt

```diff
@@ -1,6 +1,7 @@
 package org.oppia.app.home
 
 import android.os.Bundle
+import org.oppia.app.R
 import org.oppia.app.activity.InjectableAppCompatActivity
 import org.oppia.app.player.exploration.ExplorationActivity
```

Comment:
Though this is not introduced in your PR, still remove this unused import.

Reply:
Removed